mentat/conversation.py

```diff
@@ -82,7 +83,7 @@ async def display_token_count(self):
             }
             stream.send(
                 f"Included file{message[(_plural, _exceed)]} token limit"
-                f" ({tokens} / {context_size}). Truncating based on task similarity.",
+                f" ({tokens} / {context_size}).",
```

Comment:
what was this?

Reply:
We used to truncate, but we don't do that anymore. I realized it was saying this while testing this bug.